src/Cli/dotnet/commands/dotnet-add/AddCommandParser.cs

```diff
@@ -33,8 +33,6 @@ private static Command ConstructCommand()
             command.AddCommand(AddPackageParser.GetCommand());
             command.AddCommand(AddProjectToProjectReferenceParser.GetCommand());
 
-            command.SetHandler((parseResult) => parseResult.HandleMissingCommand());
```

Comment:
I removed all use of HandleMissingCommand in favor of a middleware that shows help when a command is triggered and that command doesn't have an explicit command.